script/sync_prod_to_gcp/webnode_pdf_request.py

```diff
@@ -125,6 +128,27 @@ def ping_callback(message: Message) -> None:
             message.ack()
             return
 
+        message_age: timedelta = datetime.utcnow() - message.publish_time
+        compilation_timeout = int(os.environ.get("TEX_COMPILATION_TIMEOUT_MINUTES", "30"))
+        if message_age > timedelta(minutes=compilation_timeout):
+            help_needed = os.environ.get("TEX_COMPILATION_RECIPIENT", "help@arxiv.org")
+            subject = f"TeX compilation for {paper_id}v{version} failed"
+            mail_body = f"Hello EUTA,\nTex compilation for {paper_id}v{version} has failed. Please resolve the issue.\n\nThis message is generated by a bot on arxiv-sync.serverfarm.cornell.edu.\n"
```

Comment:
EUST not EUTA.

This isn't very helpful because it doesn't tell us which node it's failing on, nor does it provide the link to review the tex log. This is better than nothing, but is it also killing the long-running tex process? we shouldn't allow them to run that long, I figure.